File: backend/api-service/Api/Api/Models/DTOs/Anime/AnimeDto.cs
using System.Text.Json;

namespace Api.Models;

public class AnimeDto
{
    public long Id { get; set; }
    public int MalId { get; set; }
    public string Url { get; set; } = string.Empty;
    public string? LargeImageUrl { get; set; } // Просто строка URL
    public string Title { get; set; } = string.Empty;
    public string? TitleEnglish { get; set; }
    public string? Type { get; set; }
    public int? Episodes { get; set; }
    public string? Status { get; set; }
    public string? Rating { get; set; }
    public double? Score { get; set; }
    public string? Synopsis { get; set; }
    public int? Year { get; set; }
}

File: mobile/application/app/src/main/java/com/example/choiceiomobile/data/models/AnimeDto.kt
package com.example.choiceiomobile.data.models

import com.google.gson.annotations.SerializedName

data class FeedResponse(
    val success: Boolean,
    val mood: String,
    val count: Int,
    val anime: List<AnimeDto>
)

data class AnimeDto(
    val id: Int,
    val malId: Int,
    val title: String,
    @SerializedName("titleEnglish") val titleEnglish: String?,
    val type: String?,
    val episodes: Int?,
    val score: Double?,
    val synopsis: String?,
    val year: Int?,
    @SerializedName("largeImageUrl") val imageUrl: String?
)